packages/cli/ete-tests/src/tests/init/init.ts

```diff
@@ -4,8 +4,9 @@ import { runFernCli } from "../../utils/runFernCli";
 
 export async function init({
     directory,
-    openApiArg
-}: { directory?: AbsoluteFilePath; openApiArg?: string } = {}): Promise<AbsoluteFilePath> {
+    openApiArg,
+    mintJsonArg
+}: { directory?: AbsoluteFilePath; openApiArg?: string; mintJsonArg?: string } = {}): Promise<AbsoluteFilePath> {
```

Comment:
nit: would probably make this accept a discriminated union of args and it can either be openapi or mintlify (cause i dont think `--openapi` and `--mintlify` together would work)

Reply:
@dsinghvi Took a slightly different path vs. a discriminated union but achieved a similar outcome. Let me know what you think!